docs/Util/InsertTags.md

```diff
@@ -16,30 +16,51 @@ the insert tag flags.
 
 See the [Contao documentation][flags] for available flags.
 
+## Date / Time
 
-### Date / Time formatting
+### Date and time formatting
 
 The `formatted_datetime` insert tag allows to format a timestamp
 using either the internal date/time formatting settings or a custom
 format.
 
 ##### Examples:
 
- 1. `{{formatted_datetime::1234::d.m.Y}}`  
+ 1. `{{formatted_datetime::1234::d.m.Y}}`
  	Formats the timestamp `1234` to Day.Month.Year.
  	Available formatting options can be found in the [PHP `date` method][date].
- 	
- 2. `{{formatted_datetime::1234::date}}`  
+
+ 2. `{{formatted_datetime::1234::date}}`
  	Formats the timestamp `1234` to the system's date format.
- 	
- 3. `{{formatted_datetime::1234::time}}`  
+
+ 3. `{{formatted_datetime::1234::time}}`
  	Formats the timestamp `1234` to the system's time format.
- 	
- 4. `{{formatted_datetime::1234::datim}}`  
+
+ 4. `{{formatted_datetime::1234::datim}}`
  	Formats the timestamp `1234` to the system's date + time format.
 
 
+### Date and time converting
+
+The `convert_dateformat` insert tag allows to convert the provided
+date/time from one format to another. It takes the following format:
 
+`{{convert_dateformat::<value>::<source_format>::<target_format>}}`
+
+The `source_format` and `target_format` can be any format from [PHP `date` method][date]
+or `date` / `datim` / `time` to take the the format from the root page settings
+(or system settings, in case not defined).
+
+##### Examples:
+
+ 1. `{{convert_dateformat::2018-11-21 10:00::datim::date}}`
+ 	Converts the provided date and time to date only `2018-11-21`.
+
+ 2. `{{convert_dateformat::21.11.2018::d.m.Y::j. F Y}}`
+ 	Converts the provided date to another format `21. November 2018`.
```

Comment:
Can we take `March` here and say that in case the page language is set to German it would say `März` so we documented that it's multilingual too?